hudi-common/src/main/java/org/apache/hudi/common/config/HoodieStorageConfig.java

```diff
@@ -90,6 +94,7 @@ public class HoodieStorageConfig extends HoodieConfig {
   public static final ConfigProperty<String> LOGFILE_DATA_BLOCK_FORMAT = ConfigProperty
       .key("hoodie.logfile.data.block.format")
       .noDefaultValue()
+      .withValidValues(AVRO_DATA_BLOCK.getId().toUpperCase(), HFILE_DATA_BLOCK.getId().toUpperCase(), PARQUET_DATA_BLOCK.getId().toUpperCase())
```

Comment:
Can these make smaller case config values unacceptable?

Reply:
Yes, but also made other changes to auto capitalize those config values when set. So should not impact anyone who sets the config with lowercase